### PR TITLE
Reskin : Fix #1971 - Any character is allowed on creation of client, user and employee

### LINKED ISSUE #1971: Reskin&oldSkin: Any character is allowed on creation of client, user and employee
Reported by Mexina at https://issues.apache.org/jira/browse/FINERACT-413

Original Description:
The system allows any character in firstname, middlename, lastname and username during creation of user/client/employee. A user of the system can enter "." in both fields and the system will allow to submit. Practically i don't think if this is correct
Reproduce this by:
Client->Create client
Fill '.' (dot) In the field for first name, middle name and last name and other mandatory fields.
The system will allow the submission successful 
You do the same in User creation and Employee creation

### PR DESCRIPTION
**Pull request by TechnicaVoyage**
Before on user creation https://demo.openmf.org/newbeta/#/createuser/:
screenshot 84
Now:
screenshot 85

Affected URLs:
https://demo.openmf.org/newbeta/#/createclient
https://demo.openmf.org/newbeta/#/createuser/
https://demo.openmf.org/newbeta/#/createemployee/

### CODE REVIEW

File: app/views/administration/createuser.html [hunk 1]
```diff
@@ -25,9 +25,12 @@
 	
 	            <div class="col-sm-3">
 	                <input type="text" id="firstname" name="firstname" ng-model="formData.firstname" class="form-control" required
-	                       late-Validate/>
+	                       ng-pattern="/(^[A-z])[A-z.'-]*/" late-Validate/>
 	            </div>
 	            <div class="col-sm-3">
+                  <span ng-show="createuserform.firstname.$invalid && createuserform.firstname.$dirty">
+                      </small>
```
Comment (gkrishnan724) [inline app/views/administration/createuser.html]: @TechnicaVoyage  You forgot to add this
```
<small class="error required" ng-show="createuserform.firstname.$error.pattern">
                         {{'validation.msg.client.name.cannot.start.specialchar.ornumber' | translate}}
</small>
```

**Comment by gkrishnan724** [comment]
@TechnicaVoyage Please keep the PR up to date.. you can just go to your current branch and rebase it

`git pull --rebase https://github.com/openMF/community-app.git new_reskin` then it will be ready for a merge I guess

**Comment by gkrishnan724** [comment]
@mbj36

**Comment by TechnicaVoyage** [comment]
@gkrishnan724 @mbj36 Done.

**Pull request merged**